### PR TITLE
Integration test setup

### PR DESCRIPTION
**Pull request by cowboy-bebug**
Changes:
- Add optional doughnut and fee_exchange param for signed_extra() in `testing/src/keyring.rs`
- Add 2 tests for simple GA transfer with & without FeeExchange
- Add empty tests so they can be implemented in the gas payment PR

### CODE REVIEW

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -111,7 +111,7 @@ parameter_types! {
 	pub const Version: RuntimeVersion = VERSION;
 }
 
-type CennznetDoughnut = prml_doughnut::PlugDoughnut<Runtime>;
+pub type CennznetDoughnut = prml_doughnut::PlugDoughnut<Runtime>;
```
Comment (jordy25519) [inline runtime/src/lib.rs:114]: We should put this in primitives like all the other types

**Review by jordy25519 — APPROVED** [review]
Nice one!

File: runtime/src/lib.rs (hunk 1 above)
Comment (jordy25519) [reply runtime/src/lib.rs:114]: can ignore this

Comment (cowboy-bebug) [reply runtime/src/lib.rs:114]: Yeah I wanted to preserve & use Runtime and stuff in it as much as possible

**Pull request merged**